client/common/js/details-table.js

```diff
@@ -1,20 +1,13 @@
-if (document.querySelector('#details-toggle-btn')) {
-	const detailsBtn = document.querySelector('#details-toggle-btn')
-	const detailsDiv = document.querySelector('#incident-details')
-
-	detailsDiv.dataset.visible = 'false'
-	detailsBtn.setAttribute('aria-expanded', 'false')
-	detailsBtn.classList.add('has-js')
-
-	detailsBtn.addEventListener('click', () => {
-		if (detailsDiv.dataset.visible === 'false') {
-			detailsDiv.dataset.visible = 'true'
-			detailsBtn.setAttribute('aria-expanded', 'true')
-			detailsBtn.innerText = 'Show Less'
-		} else if (detailsDiv.dataset.visible === 'true') {
-			detailsDiv.dataset.visible = 'false'
-			detailsBtn.setAttribute('aria-expanded', 'false')
-			detailsBtn.innerText = 'Show More'
-		}
```

Comment:
Removing this code makes the individual incident pages not function properly in smaller screens.I guess we can keep both the sections?